src/java/org/apache/cassandra/io/util/DataOutputBuffer.java

```diff
@@ -90,7 +90,7 @@ public DataOutputBuffer(ByteBuffer buffer)
     @Override
     public void flush() throws IOException
     {
-        throw new UnsupportedOperationException();
+
```

Comment:
That worries me. Can you please fire a CI run? I would do it for you but I don't have the privileges.

Reply:
Tests are https://app.circleci.com/pipelines/github/dcapwell/cassandra?branch=jmxDump, I can trigger Python dtests but unit/jvm-dtests are green.

Reply:
this behavior also matches other OutputStreams such as `java.io.ByteArrayOutputStream`, also it is an optional method in java

```
public void flush() throws IOException {
}
```

Reply:
Fair enough, no objections.